### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by jasminexie** [postcss/postcss-calc#100]
Hi! I'm using postcss-calc in one of my company projects, and we've encountered a few issues with the parser, most of which were related to custom properties, like (https://github.com/postcss/postcss-calc/issues/77, https://github.com/postcss/postcss-calc/issues/91). I found that these have been fixed in the recent refactor, but the patch has not been released.

This is an important fix for many users, so I am curious as to when there will be a release. Thank you!

**Comment by evilebottnawi** [comment]
@jasminexie hi, in my todo list, a lot of work and nobody want to help

**Comment by jasminexie** [comment]
> @jasminexie hi, in my todo list, a lot of work and nobody want to help

Thank you for the hard work on this repo. If there's anything the community can do I'll be happy to help. I'll be watching this for updates.

**Comment by evilebottnawi** [comment]
@jasminexie we have other bugs https://github.com/postcss/postcss-calc/issues, feel free to pick up any and send a PR

**Comment by jasminexie** [comment]
Hi, I've looked through the issues and run a few tests with the current master branch. #91, #89, #77 are all fixed with the recent refactor. 

I've run a local test with the latest master of `postcss-calc` and the latest version of `postcss-custom-variables`. #87 reduces correctly, and I believe is fixed.

Most of the issues mentioned above already have test cases covering them, but I can add more tests if necessary.

I don't think #92 is a bug, and the remaining open issues (#67 and #62) are all feature requests. There is an open PR for #62 that needs to be rebased against the latest master.

In short, I don't think the current master branch has any other reported bugs. Unless you're waiting for feature PRs, I believe this project is ready for a release. 😄

**Comment by evilebottnawi** [comment]
@jamiebuilds Thanks

**Comment by evilebottnawi** [comment]
Please look here https://github.com/cssnano/cssnano/issues/833, postcss-calc is part of cssnano

**Comment by alecmarcus** [comment]
I've run into #87, but commenting here because it seems more relevant. Is there anything stopping the next release from `master`? Would be great to have the fix for all these issues. Anything I can do to help?

**Comment by Semigradsky** [comment]
@jasminexie @alecmarcus 
https://github.com/postcss/postcss-calc/releases/tag/7.0.2
